lib/ansible/modules/network/cnos/cnos_static_route.py

```diff
@@ -0,0 +1,298 @@
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Module to work on Link Aggregation with Lenovo Switches
+# Lenovo Networking
+#
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+---
+module: cnos_static_route
+version_added: "2.8"
+author: "Anil Kumar Muraleedharan (@amuraleedhar)"
+short_description: Manage static IP routes on Lenovo CNOS network devices
+description:
+  - This module provides declarative management of static
+    IP routes on Lenovo CNOS network devices.
+notes:
+  - Tested against CNOS 10.10.1
+options:
+  prefix:
+    description:
+      - Network prefix of the static route.
+  mask:
+    description:
+      - Network prefix mask of the static route.
+  next_hop:
+    description:
+      - Next hop IP of the static route.
+  interface:
+    description:
+      - Interface of the static route.
+  description:
+    description:
+      - Name of the static route
+    aliases: ['description']
+  admin_distance:
+    description:
+      - Admin distance of the static route.
+    default: 1
+  tag:
+    description:
+      - Set tag of the static route.
+  aggregate:
+    description: List of static route definitions.
+  state:
+    description:
+      - State of the static route configuration.
+    default: present
+    choices: ['present', 'absent']
+"""
+
+EXAMPLES = """
+- name: configure static route
+  cnos_static_route:
+    prefix: 10.241.107.0
+    mask: 255.255.255.0
+    next_hop: 10.241.106.1
+
+- name: configure ultimate route with name and tag
+  cnos_static_route:
+    prefix: 10.241.107.0
+    mask: 255.255.255.0
+    interface: Ethernet1/13
+    description: hello world
+    tag: 100
+
+- name: remove configuration
+  cnos_static_route:
+    prefix: 10.241.107.0
+    mask: 255.255.255.0
+    next_hop: 10.241.106.0
+    state: absent
+
+- name: Add static route aggregates
+  cnos_static_route:
+    aggregate:
+      - { prefix: 10.241.107.0, mask: 255.255.255.0, next_hop: 10.241.105.0 }
+      - { prefix: 10.241.106.0, mask: 255.255.255.0, next_hop: 10.241.104.0 }
+
+- name: Remove static route aggregates
+  cnos_static_route:
+    aggregate:
+      - { prefix: 10.241.107.0, mask: 255.255.255.0, next_hop: 10.241.105.0 }
+      - { prefix: 10.241.106.0, mask: 255.255.255.0, next_hop: 10.241.104.0 }
+    state: absent
+"""
+
+RETURN = """
+commands:
+  description: The list of configuration mode commands to send to the device
+  returned: always
+  type: list
+  sample:
+    - ip route 10.241.107.0 255.255.255.0 10.241.106.0
+"""
+from copy import deepcopy
+from re import findall
+
+try:
+    import ipaddress
```

Comment:
can you use `ansible.module_utils.compat.ipaddress` instead?

Reply:
Done